deploy/template.yaml

```diff
@@ -1458,79 +1458,6 @@ Resources:
         TargetGroupName: !GetAtt LoadBalancerListenerTargetGroupECS.TargetGroupName
         VpcId: !Sub ${VpcStackName}-VpcId
 
-  # Metrics for frontend-vital-signs
```

Comment:
What about the installed component in package.json and the changes the front-end. A revert might not work here so would have to do this manually

Reply:
We are not removing the package from the frontend. Just the MetricFilter from CloudFormation.